ctwrap/simulation.py

```diff
@@ -21,25 +22,25 @@
 
 class Simulation(object):
     """The Simulation class wraps modules into an object.
-
     Attributes:
         data (dict): dictionary of pandas.DataFrame objects
-
     Required module functions 'run' and 'defaults' are passed through
     """
 
     def __init__(self, module, output=None):
         """Constructor for `Simulation` object.
-
         Arguments:
-           module (module): handle to module running the simulation
+           module (module):  handle name or handle to module running the simulation
```

Comment:
call this: "module or name of module running the simulation"
I think the pep 484 convention would be `Union[object, str]`